packages/cozy-stack-client/src/AppCollection.js

```diff
@@ -37,31 +35,6 @@ class AppCollection {
       next: false
     }
   }
-
-  find(...args) {
-    console.warn(
-      `find() method for doctype '${
-        this.doctype
-      }' relies internally on DocumentCollection.find() and may not fetch all expected data.`
-    )
-    return DocumentCollection.prototype.find.call(this, ...args)
-  }
```

Comment:
You will lose the `console.warn`. If you want to keep it, you can use `super.find()`